src/components/MultiHopTrade/hooks/useAccountIds.tsx

```diff
@@ -19,31 +13,8 @@ export const useAccountIds = (): {
   const dispatch = useAppDispatch()
 
   // Default sellAssetAccountId selection
-
   const sellAssetAccountId = useAppSelector(selectFirstHopSellAccountId)
-  const sellAssetAccountNumberFilter = useMemo(
-    () => ({ accountId: sellAssetAccountId }),
-    [sellAssetAccountId],
-  )
-  const sellAssetAccountNumber = useAppSelector(state =>
-    selectAccountNumberByAccountId(state, sellAssetAccountNumberFilter),
-  )
-
-  // Default buyAssetAccountId selection
-
-  const accountIdsByAccountNumberAndChainId = useAppSelector(
-    selectAccountIdByAccountNumberAndChainId,
-  )
-  const inputBuyAsset = useAppSelector(selectInputBuyAsset)
-  const maybeMatchingBuyAccountId =
-    accountIdsByAccountNumberAndChainId[sellAssetAccountNumber as number]?.[inputBuyAsset?.chainId]
-
-  // We always default the buy asset account to be synchronized with the sellAssetAccountNumber
-  // - if this isn't possible, i.e there is no matching account number on the buy side, we default to the highest balance
-  // - if this was to fail for any reason, we default to the first account number as a default
-  const buyAssetAccountId = useAppSelector(state =>
-    selectLastHopBuyAccountId(state, { accountId: maybeMatchingBuyAccountId }),
-  )
```

Comment:
all of this logic has been updated and shifted into the selector `selectLastHopBuyAccountId`

Reply:
👀